lte/gateway/release/BUILD.bazel

```diff
@@ -10,9 +10,8 @@
 # limitations under the License.
 
 """
-Artifacts in this package should be build with the production
-configuration:
-bazel build lte/gateway/release:sctpd_deb_pkg --config=production
+Definitions of the magma and sctpd debian artifacts.
+See "//lte/gateway/release:release_build" on how artifacts can be created.
```

Comment:
nitpick:  Given that that target lives in this package, we could just reference the package-local name ("look at the the comment in the "release_build" target for information on how to build the Magma deb package")

Reply:
:+1: changed